pkg/parser/node.go

```diff
@@ -412,27 +410,24 @@ func (n *Node) compile(pctx *UnixParserCtx) error {
 	}
 	/* compile leafs if present */
 	if len(n.SuccessNodes) > 0 {
-		for idx, _ := range n.SuccessNodes {
+		for idx := range n.SuccessNodes {
```

Comment:
I am very surprised at this one. Do you have some hint on this ?

Reply:
Because we only care about the index in the iteration, not the value itself